backend/app/db/database_service.py

```diff
@@ -59,53 +56,45 @@ async def insert_genres_to_cache(genres: dict) -> None:
+                    )
+                    if provider.get(country_code)
+                ],
+                providers=[
+                    provider.get(country_code)
+                    for provider in unique_list(
+                        media.flatrate_providers, media.free_providers
+                    )
+                    if provider.get(country_code)
+                ],
+            ).dict()
+            for media in media
+        ]
+    )
+
+
+def init_meilisearch_indexing():
     """MeiliSearch indexing from Postgres DB"""
     db = database.SessionLocal()
-    all_media_length = count_all_media(db)
-    chunk_loops = ceil(all_media_length / chunk_size)  # Will always round up
+    media: list = get_all_media(db)
 
-    with engine.connect() as conn:
-        media_stream = conn.execution_options(stream_results=True).execute(
-            Media.__table__.select()
-        )
-        print(
-            f"MeiliSearch is about to index "
-            f"{len(get_settings().supported_country_codes)} countries"
-        )
-        with tqdm(total=chunk_loops, desc="Queuing up documents for indexing") as pbar:
-            while chunk := media_stream.fetchmany(chunk_size):
-                for country_code in get_settings().supported_country_codes:
-                    client.index(f"media_{country_code}").add_documents(
-                        [
-                            schemas.Media(
-                                id=media.id,
-                                title=media.title,
-                                original_title=media.original_title,
-                                overview=media.overview,
-                                release_date=media.release_date,
-                                genres=media.genres,
-                                poster_path=media.poster_path,
-                                popularity=media.popularity,
-                                provider_names=[
-                                    provider.get(country_code).get("provider_name")
-                                    for provider in unique_list(
-                                        media.flatrate_providers, media.free_providers
-                                    )
-                                    if provider.get(country_code)
-                                ],
-                                providers=[
-                                    provider.get(country_code)
-                                    for provider in unique_list(
-                                        media.flatrate_providers, media.free_providers
-                                    )
-                                    if provider.get(country_code)
-                                ],
-                            ).dict()
-                            for media in chunk
-                        ]
-                    )
-                pbar.update(1)
+    for country_code in tqdm(get_settings().supported_country_codes):
```

Comment:
Could we have a description for this step? Right now it looks like this:
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/50198099/159013383-e8329595-a025-45d8-8f53-66e800e4cd29.png">

Reply:
Done